roles/build-release/vars/main.yml

```
@@ -0,0 +1,13 @@
+---
+# Note: the value is either _ansible_base_version or _ansible_core_version depending on the version
+# ex: https://github.com/ansible-community/ansible-build-data/blob/main/4/ansible-4.4.0.deps
+# Variables that start with `_` are intermediate variables that are `register`ed in the playbook.
+antsibull_expected_ansible_core: "{{ _deps['_ansible_core_version'] | default(_deps['_ansible_base_version']) }}"
+antsibull_pip_bin: "{{ antsibull_ansible_venv }}/bin/pip"
+antsibull_pip_pkgs: "{{ _pip_pkgs['packages'][antsibull_pip_bin] }}"
+ansible_version_pypi: "{{ antsibull_pip_pkgs['ansible'][0]['version'] }}"
+_installed_ansible_core: "{{ antsibull_pip_pkgs['ansible-core'][0]['version'] }}"
```

How about always using the prefix `antsibull_ansible_venv` for everything that's in antsibull's ansible venv?
```suggestion
antsibull_ansible_venv_pip_bin: "{{ antsibull_ansible_venv }}/bin/pip"
antsibull_ansible_venv_pip_pkgs: "{{ _pip_pkgs['packages'][antsibull_pip_bin] }}"
antsibull_ansible_venv_installed_ansible_version: "{{ antsibull_pip_pkgs['ansible'][0]['version'] }}"
antsibull_ansible_venv_installed_ansible_core_version: "{{ antsibull_pip_pkgs['ansible-core'][0]['version'] }}"
```

---

Fine for me. @dmsimard what do you think?

---

See https://github.com/ansible-community/antsibull/pull/432/commits/028b35f5ce59f0443a8c1897fc13425cfb7287fd. I'm thinking it would make sense to completely remove the `venv` prefix, but I'll defer to you and @dmsimard.